awslambda/project.json

```diff
@@ -0,0 +1,39 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "compilerName": "fsc",
+    "emitEntryPoint": false,
+    "compile": "*.fs",
+    "debugType": "portable",
+    "outputName": "<%= namespace %>"
```

Comment:
Cool, I didn't actually know that, was just trying to change it as little as possible.